storage/rocksdb/ha_rocksdb.cc

```diff
@@ -8260,16 +8302,37 @@ int ha_rocksdb::update_sk(const TABLE *const table_arg, const Rdb_key_def &kd,
 
   const bool store_row_debug_checksums = should_store_row_debug_checksums();
 
+  m_sk_tails.clear();
+  // Reserve space for index flag fields
```

Comment:
This should happen in pack_record() instead. unpack_info is generated there. The TTL field should also be added there.